### DISCUSSION TITLE
How to refresh Dialog

### DISCUSSION THREAD

**Discussion by ruzicao** [serenity-is/Serenity#5944] [Help]
I added Refresh button to dialog form. Since the content of the datebase is updating every five second, I want refresh content on opened dialog, too. How can I do it? 
I tried with:   
 var refreshBtn = $('.refresh-button');

            refreshBtn.click(function refresh() {
            this.reloadById(entity.Id);) ,

 but it doesn't work.

**Comment by luongnv6** [comment]
just use:
this.reloadById(); 
for click event
